src/utils/crypto.js

```diff
@@ -65,7 +65,25 @@ const Crypto = {
         let signature = Axlsign.sign(privateKeyBytes, dataBytes, SecureRandom.default.randomUint8Array(64));
         return Base58.encode(signature);
     },
-    isValidTransactionSignature: function (dataBytes, signature, publicKey) {
+    isValidTransactionSignature: function (transaction, signature, publicKey) {
+        let dataBytes = transaction.toBytes();
```

Comment:
Need to check `transaction` is null or not. And need to check type as well, or it would not have `toBytes` method.